drip/utils.py

```diff
@@ -192,10 +195,23 @@ def get_simple_fields(Model, **kwargs):
 
 
 def get_user_model():
-    # handle 1.7 and back
+    """
+    Support the case when the users need in the drip campaign are not from
+    Auth User model
+    """
     try:
-        from django.contrib.auth import get_user_model as django_get_user_model
-        User = django_get_user_model()
-    except ImportError:
-        from django.contrib.auth.models import User
-    return User
+        return django_apps.get_model(
+            getattr(settings, 'DRIP_CAMPAIGN_USER_MODEL',
+                    getattr(settings, 'c', 'auth.User')),
+            require_ready=False
+        )
+    except ValueError:
+        raise ImproperlyConfigured(
+            "DRIP_CAMPAIGN_MODEL must be of the form 'app_label.model_name'"
+        )
+    except LookupError:
+        raise ImproperlyConfigured(
+            "DRIP_CAMPAIGN_MODEL"
+            " refers to model '%s' that has not been installed"
```

Comment:
Please change this to keep consistency in the project:
In Django drip the defined way by the team is: 
```
'DRIP_CAMPAIGN_MODEL refers to model {user_model} that has not been installed'.format(user_model=settings.AUTH_USER_MODEL)
```

We defined to use `format`.